Actions/EventAction.h

```diff
@@ -51,6 +51,7 @@ class EventAction : public G4UserEventAction
 
   bool is2gRec;
   bool is3gRec;
+  int fEventID;
```

Comment:
I am not sure if there is a need to store event ID here, in `EventAction` class you can access event and its ID from Geant4 class by `GetEventID()`.